Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -200,62 +145,64 @@ CGPathRef CGPathCreateCopy(CGPathRef path) {
 }
 
 /**
- @Status Interoperable
+@Status Interoperable
+@Notes
 */
```

Comment:
Not particularly no.

---
In reply to: [83931964](https://github.com/Microsoft/WinObjC/pull/1138#discussion_r83931964) [](ancestors = 83931964)